build.zig

```diff
@@ -794,6 +789,7 @@ fn buildLibDawnNative(b: *Build, step: *std.build.CompileStep, options: Options)
         include("libs/dawn/out/Debug/gen/src"),
     });
     if (options.d3d12.?) try flags.appendSlice(dawn_d3d12_flags);
+    if (options.vulkan.?) try flags.appendSlice(&.{include("libs/dawn/third_party/vulkan-deps/vulkan-headers/src/include")});
```

Comment:
we can't depend on Dawn's vulkan_headers and should remove them.

I will probably need to update https://github.com/hexops/vulkan-headers